src/main/kotlin/com/software/modsen/carservice/config/MapperConfig.kt

```diff
@@ -0,0 +1,16 @@
+package com.software.modsen.carservice.config
+
+import org.modelmapper.ModelMapper
+import org.modelmapper.convention.MatchingStrategies
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class MapperConfig {
+    @Bean
+    fun modelMapper(): ModelMapper {
+        val modelMapper = ModelMapper()
+        modelMapper.configuration.matchingStrategy = MatchingStrategies.STRICT
+        return ModelMapper()
```

Comment:
Check scope function .apply() - you can make your code very readable

Reply:
Used .apply()